salt/modules/postgres.py

```diff
@@ -199,6 +199,8 @@ def _run_initdb(name,
         password=None,
         encoding='UTF8',
         locale=None,
+        waldir=None,
```

Comment:
Can you move these new kwargs to the end of the list? We don't want to cause an API change in case users are passing positional args.